charts/lnd/values.yaml

```diff
@@ -106,7 +106,7 @@ affinity:
             - bitcoind
         topologyKey: kubernetes.io/hostname
 
-walletPassword: 12345678
+walletPassword: ""
```

Comment:
I guess an empty string would be defined as password is not set? (versus password is an empty string)? have you tested it?

Reply:
> I guess an empty string would be defined as password is not set? (versus password is an empty string)? have you tested it?

Yes I have tested it. Empty string returns false inside an `if`